### PR TITLE
[Doc] Add more cloud provider examples to shared-data

### PR DESCRIPTION
**Pull request by EsoragotoSpirit**
Signed-off-by: EsoragotoSpirit <wanglichen@starrocks.com>

### CODE REVIEW

File: administration/deploy_shared_data.md [hunk 1]
```diff
@@ -116,12 +112,88 @@ StarRocks 存算分离集群的部署方式与普通 StarRocks 集群的部署
...
+
+  ```YAML
+  run_mode = shared_data
+  cloud_native_meta_port = <meta_port>
+  cloud_native_storage_type = S3
+  aws_s3_path = <s3_path>
+  aws_s3_region = <region>                 # 例如：us-east-1
+  aws_s3_endpoint = <endpoint_url>         # 例如：http://172.26.xx.xxx:39000
+  aws_s3_access_key = <access_key>
+  aws_s3_secret_key = <secrete_key>
```
Comment (evelynzhaojie) [inline administration/deploy_shared_data.md:182]: secrete --> secret

File: administration/deploy_shared_data.md [hunk 2]
```diff
@@ -116,12 +112,88 @@ StarRocks 存算分离集群的部署方式与普通 StarRocks 集群的部署
...
+
+  ```YAML
+  run_mode = shared_data
+  cloud_native_meta_port = <meta_port>
+  cloud_native_storage_type = S3
+  aws_s3_path = <s3_path>
+  aws_s3_region = <region>                 # 例如：cn-beijing
+  aws_s3_endpoint = <endpoint_url>         # 例如：https://tos-s3-cn-beijing.ivolces.com
+  aws_s3_access_key = <access_key>
+  aws_s3_secret_key = <secrete_key>
```
Comment (evelynzhaojie) [inline administration/deploy_shared_data.md:169]: secrete --> secret

File: administration/deploy_shared_data.md [hunk 3]
```diff
@@ -116,12 +112,88 @@ StarRocks 存算分离集群的部署方式与普通 StarRocks 集群的部署
...
+
+  ```YAML
+  run_mode = shared_data
+  cloud_native_meta_port = <meta_port>
+  cloud_native_storage_type = S3
+  aws_s3_path = <s3_path>
+  aws_s3_region = <region>                 # 例如：ap-beijing
+  aws_s3_endpoint = <endpoint_url>         # 例如：https://cos.ap-beijing.myqcloud.com
+  aws_s3_access_key = <access_key>
+  aws_s3_secret_key = <secrete_key>
```
Comment (evelynzhaojie) [inline administration/deploy_shared_data.md:156]: secrete --> secret

File: administration/deploy_shared_data.md [hunk 4]
```diff
@@ -116,12 +112,88 @@ StarRocks 存算分离集群的部署方式与普通 StarRocks 集群的部署
...
+
+  ```YAML
+  run_mode = shared_data
+  cloud_native_meta_port = <meta_port>
+  cloud_native_storage_type = S3
+  aws_s3_path = <s3_path>
+  aws_s3_region = <region>                 # 例如：cn-north-4
+  aws_s3_endpoint = <endpoint_url>         # 例如：https://obs.cn-north-4.myhuaweicloud.com
+  aws_s3_access_key = <access_key>
+  aws_s3_secret_key = <secrete_key>
```
Comment (evelynzhaojie) [inline administration/deploy_shared_data.md:143]: secrete --> secret

File: administration/deploy_shared_data.md [hunk 5]
```diff
@@ -116,12 +112,88 @@ StarRocks 存算分离集群的部署方式与普通 StarRocks 集群的部署
...
+
+  ```YAML
+  run_mode = shared_data
+  cloud_native_meta_port = <meta_port>
+  cloud_native_storage_type = S3
+  aws_s3_path = <s3_path>
+  aws_s3_region = <region>                 # 例如：cn-zhangjiakou
+  aws_s3_endpoint = <endpoint_url>         # 例如：https://oss-cn-zhangjiakou-internal.aliyuncs.com
+  aws_s3_access_key = <access_key>
+  aws_s3_secret_key = <secrete_key>
```
Comment (evelynzhaojie) [inline administration/deploy_shared_data.md:130]: secrete --> secret

Comment (evelynzhaojie) [reply administration/deploy_shared_data.md:130]: OSS，以及其他的云厂商参数都是 S3 吗，感觉挺奇怪的

**Review by evelynzhaojie — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Comment by EsoragotoSpirit** [comment]
@mergify backport 3.0

**Pull request merged**